app/js/components/manageApps/ManageApps.jsx

```diff
@@ -360,6 +360,44 @@ export default class ManageApps extends React.Component {
     });
   }
 
+  handleOWAInstallation(downloadUri) {
+    const applicationDistribution = location.href.split('/')[2];
+    const url = location.href.split('/')[3];
+    const apiBaseUrl = `/${applicationDistribution}/${url}/ws/rest`;
+    this.requestUrl = '/owa/installapp';
+    const urlObject = {
+      urlValue: downloadUri
+    };
+    const moduleName = downloadUri
+      .split('/')[5]
+      .split('=')[1]
+      .split('-')[0];
+    return (e) => {
+      e.preventDefault();
+      axios({
+        url: `https:/${apiBaseUrl}${this.requestUrl}`,
```

Comment:
Checkout how `urlPrefix` has been in building urls for other api calls in the component